build/configs/artik05x/artik05x_download.sh

```diff
@@ -188,7 +188,7 @@ erase()
 
 	# Download all binaries using openocd script
 	pushd ${OPENOCD_DIR_PATH} > /dev/null
-	${OPENOCD} -f ${CFG_FILE} -s ${SCRIPTS_PATH} -c "flash_erase_part ${PART_NAME};	exit" || exit 1
+	${OPENOCD} -f ${CFG_FILE} -s ${SCRIPTS_PATH} -c "flash_protect off; flash_erase_part ${PART_NAME};	exit" || exit 1
```

Comment:
Do we require that extra "TAB" before exit?. I think one space is good

Reply:
Not necessary. Removed.